descriptors/ModuleDescriptor-template.json

```diff
@@ -176,7 +180,9 @@
         "invoice.invoice-lines.item.put",
         "invoice.invoice-lines.item.delete",
         "invoice.invoice-number.item.get",
-        "invoice.invoice-number.item.post"
+        "invoice.invoice-number.item.post",
+        "invoice-storage.invoice-line-number.get",
```

Comment:
invoice.all only must the permissions that are provided by the module, invoice-storage permissions are not required here

Reply:
Got it. Removed.